pkg/cmd/root/root.go

```diff
@@ -38,6 +39,7 @@ func NewCmdRoot(f *factory.Factory) (*cobra.Command, error) {
 	cmd.AddCommand(pipelineCmd.NewCmdPipeline(f))
 	cmd.AddCommand(useCmd.NewCmdUse(f))
 	cmd.AddCommand(versionCmd.NewCmdVersion(f))
+	cmd.AddCommand(clusterCmd.NewCmdCluster(f))
```

Comment:
Can you keep these sorted alphabetically

Reply:
Done